types/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "target": "es6",
     "lib": ["es6", "dom"],
     "module": "ES6",
+    "types": [],
```

Comment:
We are removing node types from types checks if we do not do it we will get:

`../node_modules/@types/node/globals.d.ts(72,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type '{ new (): AbortSignal; prototype: AbortSignal; timeout(milliseconds: number): AbortSignal; }', but here has type '{ new (): AbortSignal; prototype: AbortSignal; }'.`

In general it does not make sense to have types: ['node'] (default) because it is browser running project.